reference/extensions/mapper_selectors.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: page
+title: MapperSelectors Extension
+---
+
+The `MapperSelectors` extension adds `Controller#set_selectors`, which sets selectors in the controller's `identity_map` that ensure the fields returned from `Controller#expanded_fields` (from the [`FieldExpansion` extension](./field_expansion)) are loaded for a given model when `identity_map.load(model)` is called.
+
+This will `select` any required database columns, and `track` any relevant associations.
```

Comment:
Seems like that's the important thing to say. The system selects any required database columns already, what this extension does is subtract columns that are not required.